test/lib/usd/translators/CMakeLists.txt

```diff
@@ -2,6 +2,7 @@ set(TARGET_NAME IMPORT_EXPORT_TEST)
 
 set(TEST_SCRIPT_FILES
     testUsdExportAsClip.py
+    testUsdExportAnimation.py
```

Comment:
Super nit-picky, but bump this one up before testUsdExportAsClip.py.

Reply:
Ah yeah, guess it should be alphabetical. Done